android/src/toga_android/widgets/webview.py

```diff
@@ -28,9 +28,20 @@ class WebView(Widget):
 
     def create(self):
         self.native = A_WebView(self._native_activity)
+        try:
+            from .webview_static_proxy import TogaWebClient
+
+            client = TogaWebClient(self)
+        except BaseException:
+            client = WebViewClient()
+            msg = "chaquopy.defaultConfig.staticProxy"
+            msg += '("toga_android.widgets.webview_static_proxy") '
+            msg += 'missing in pyproject.toml section "build_gradle_extra_content"\n'
+            msg += "on_navigation_starting handler is therefore not available"
```

Comment:
A multiline-string would be a better option here; see how GTK's webview handles similar long strings.

However, it would also be preferable to only output this warning if the user actually *has* an on_navigation_starting handler. Outputting the message when the handler is *set* would make more sense to me.